packages/protocol/contracts/governance/Governance.sol

```diff
@@ -615,30 +653,66 @@ contract Governance is
    * @notice Approves the hash of a hotfix transaction(s).
    * @param hash The abi encoded keccak256 hash of the hotfix transaction(s) to be approved.
    */
-  function approveHotfix(bytes32 hash) external hotfixNotExecuted(hash) onlyApprover {
-    hotfixes[hash].approved = true;
-    emit HotfixApproved(hash);
+  function approveHotfix(
+    bytes32 hash
+  ) external hotfixNotExecuted(hash) onlyApproverOrSecurityCouncil {
+    if (msg.sender == approver) {
+      hotfixes[hash].approved = true;
+    } else {
+      if (isL2()) {
```

Comment:
This condition should be in root of this function. 

if (msg.sender == approver) shoudn't be considered in L2 anymore